composer.json

```diff
@@ -12,11 +12,11 @@
     ],
     "require": {
         "php":                      ">5.3.0",
-        "symfony/framework-bundle": "~2.1|~3.0",
         "winzou/state-machine":     "~0.3"
     },
     "require-dev": {
-        "phpspec/phpspec": "~2.0"
+        "phpspec/phpspec": "~2.0",
+        "symfony/framework-bundle": "~2.1|~3.0|^4.0"
```

Comment:
Actually I don't understand why we cannot keep `symfony/framework-bundle` in the `require` section?
`symfony/skeleton` requires `symfony/framework-bundle` and `symfony/symfony` replaces `symfony/framework-bundle`, so it should be fine, right?

Reply:
Indeed, I remeember it didn't work at the time of the PR, composer saif it had a conflict. 

But I tried again and it works now. So I moved back the requirement into `require`. 
Travis build should pass in ~45min.

Ben